CHANGELOG.md

```diff
@@ -1,5 +1,51 @@
 # Changelog
 
+## 🍕 0.4.0
+
+  - ### ✨ Features
+
+    - **Add config file for configuring include / exclude of files to template - [xortive], [pull/174]**
+        
+        Adds support for the [cargo-generate.toml](https://github.com/ashleygwilliams/cargo-generate/blob/master/README.md#include--exclude) file, which allows templates to configure which files should be processed, either using a whitelist method (`include`), or a blacklist method (`exclude`). When both `include` and `exclude` are present, `include` will be preferred. This mirrors [similar `include`/`exclude` behavior in `cargo`](https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields-optional), and uses some of the same code.
+
+        ```toml
+        [template]
+        include = ["Cargo.toml"]
+        # include and exclude are exclusive, if both appear we will use include
+        exclude = ["*.c"]
+        ```
+       
+        [xortive]: https://github.com/xortive
+        [pull/174]: https://github.com/ashleygwilliams/cargo-generate/pull/174
+
+    - **Filenames can now contains template tag(s) - [k0pernicus], [pull/161]**
```

Comment:
this is not in the release anymore! make sure it's not in the README!

Reply:
Revert PR got rid of it